mlflow/tensorflow.py

```diff
@@ -475,6 +472,11 @@ def on_train_end(self, logs=None):  # pylint: disable=unused-argument
             epsilon = opt._epsilon if type(opt._epsilon) is float \
                 else tensorflow.keras.backend.eval(opt._epsilon)
             try_mlflow_log(mlflow.log_param, 'epsilon', epsilon)
+
+    def on_epoch_end(self, epoch, logs=None):
+        pass
+
+    def on_train_end(self, logs=None):  # pylint: disable=unused-argument
```

Comment:
Same thing here with moving the model summary to the beginning of training

Reply:
Done!